src/govuk/components/pagination/pagination.yaml

```diff
@@ -0,0 +1,198 @@
+params:
+- name: classes
+  type: string
+  required: false
+  description: The classes that you want to add to the pagination.
+- name: previous
+  type: object
+  required: false
+  description: A link to the previous page, if there is one.
+  params:
+  - name: text
+    type: string
+    required: false
+    description: The previous page link text. Defaults to 'Previous page', where 'page' is visually hidden.
+  - name: href
+    type: string
+    required: true
+    description: The previous page URL.
+- name: next
+  type: object
+  required: false
+  description: A link to the next page, if there is one.
+  params:
+  - name: text
+    type: string
+    required: false
+    description: The next page link text. Defaults to 'Next page', where 'page' is visually hidden.
+  - name: href
+    type: string
+    required: true
+    description: The next page URL.
+- name: items
+  type: array
+  required: true
+  description: Array of link objects.
+  params:
+  - name: type
+    type: string
+    required: false
+    description: Set to 'elipses' to indicate where a list has been truncated.
+  - name: selected
+    type: boolean
+    required: false
+    description: Set to 'true' to indicate the current page.
+  - name: text
+    type: string
+    required: false
+    description: The link text - usually a page number.
+  - name: href
+    type: string
+    required: false
+    description: The link URL.
+- name: results
+  type: object
+  required: false
+  description: Optional text describing the total number of results and which ones are on the current page.
+  params:
+  - name: from
+    type: string
+    required: true
+    description: Number of first result on page.
+  - name: to
+    type: string
+    required: true
+    description: Number of last result on page.
+  - name: count
+    type: string
+    required: true
+    description: Total number of results.
+  - name: type
+    type: string
+    required: false
+    description: Used to describe the component to screen-readers and in the results text. Defaults to 'results'.
+
+examples:
+- name: default
+  data:
+    previous:
+      href: '/page=1'
```

Comment:
This is picky—and I know it's what the MOJ Pattern Library does too—but I'd personally prefer these to be a more representative URL, like `/?page=1`. I worry that because `/page=1` isn't a common URL pattern, it might look like it has some special meaning.

Reply:
Thanks - actually, there's another issue too, which is that if you use these examples directly in the Design System website, it fails our link checker. For this reason, all the examples in the design system currently use `/#`. I think, for that reason, I'll update them to # - but perhaps a future update would be to adjust our link checker